system/filesystem.py

```diff
@@ -63,6 +63,50 @@
 - filesystem: fstype=ext4 dev=/dev/sdb1 opts="-cc"
 '''
 
+def _get_dev_size(dev, module):
+    """ Return size in bytes of device. Returns int """
+    blockdev_cmd = module.get_bin_path("blockdev", required=True)
+    rc, devsize_in_bytes, err = module.run_command("%s %s %s" % (blockdev_cmd, "--getsize64", dev))
+    return int(devsize_in_bytes)
+
+
+def _get_fs_size(fssize_cmd, dev, module):
```

Comment:
Whole function's first level of indentation is wrong - should be 4 spaces, now 8.